feature-gate/program/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! Feature Gate program
+
+#![allow(clippy::integer_arithmetic)]
```

Comment:
Unless I'm missing something, I don't see a need to allow integer arithmetic in this program. Let's be tough!